### PR TITLE
e2e-Joiner Test Scenarios

### PR DESCRIPTION
**Pull request by itsmekumari**
@sawantpritam Please review.

### CODE REVIEW

File: e2e-test/src/test/features/JoinerErrorScenarios.feature [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    When Select plugin: "File" from the plugins list as: "Source"
+    When Select plugin: "File" from the plugins list as: "Source"
+    And Expand Plugin group in the LHS plugins list: "Analytics"
+    When Select plugin: "Joiner" from the plugins list as: "Analytics"
+    Then Connect plugins: "File" and "Joiner" to establish connection
+    Then Connect plugins: "File2" and "Joiner" to establish connection
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "File" from the plugins list as: "Sink"
+    Then Connect plugins: "Joiner" and "File3" to establish connection
+    And Click plugin property: "pipeline-clean-up-graph-control" button
```
Comment (sawantpritam) [inline e2e-test/src/test/features/JoinerErrorScenarios.feature]: And -> Then

Comment (sawantpritam) [inline e2e-test/src/test/features/JoinerErrorScenarios.feature]: Use property from data-cy attributes file.

**Review by sawantpritam — CHANGES_REQUESTED** [review]

File: e2e-test/src/test/features/JoinerErrorScenarios.feature (hunk 1 above)
Comment (itsmekumari) [reply e2e-test/src/test/features/JoinerErrorScenarios.feature]: Change done.

Comment (itsmekumari) [reply e2e-test/src/test/features/JoinerErrorScenarios.feature]: Change done.

**Pull request closed without merging**